src/docs/contributing.md

```diff
@@ -0,0 +1,134 @@
+   - `release/*` for [releases](/docs/releasing) (administrators only)
+
+4. **Write clear, helpful and descriptive commit messages.**
+
+   1. **Use imperative and write in English,** e.g. *Update dependencies* or
+      *Claim support for controlled components only*.
+   2. **If an issue exists for your changes, append the issue number** in
+      parentheses to the end of the commit message, e.g. *Update dependencies
+      (#261)*.
+   3. Optionally use Markdown code blocks to emphasize, e.g.
+      *Create `ScrollView` component (#53)*.
+
+5. **Write clear, helpful and descriptive PR names.**
+
+   1. **All rules for commit messages apply** also for PR names.
+   2. **Always check that PR name meets the requirements** above because **PR
+      names are used in changelog**. GitHub automatically truncates long PR
+      names and picks up branch name for multi-commit PRs, so it's necessary to
+      make sure the PR name is what we want to have in the changelog.
+   3. **If an issue exists for your changes, append this text to PR
+      description** (the topmost comment in the PR) in order for the issue to be
+      closed automatically once the PR is merged: `Closes #<ISSUE NUMBER>`. You
+      will know the issue is linked correctly when it appears in the *Linked
+      issues* section of the PR. (Having the issue mentioned in commit message
+      and/or PR name does *not* have this effect.)
+   4. **If there is no issue for your changes, please add your PR to `The
+      Board`** GitHub project in the *Projects* section of the PR. The correct
+      board column will be selected automatically. This helps us keep track of
+      what is in development.
+
+Pull requests are labelled automatically. You can add more labels to better
+qualify the nature of the change — in such case, it will be included in all
+corresponding changelog groups. Or use the `skip changelog` label to exclude a
+pull request from the changelog.
+
+## Package Linking
+
+The best way for development of React UI is to link `react-ui` into your
+application with `npm link` so you can see it in action.
+
+1. In React UI repository, run `npm link`
+2. In your application, run `npm link @react-ui-org/react-ui`
+
+To prevent
+[Invalid Hook Call Warning](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react)
+when React UI is linked, add the following code to your app's Webpack config:
+
+```js
+const path = require('path');
+
+module.exports = {
+  resolve: {
+    alias: {
+      react: path.resolve('./node_modules/react'),
+      'react-dom': path.resolve('./node_modules/react-dom'),
+    },
+  },
+};
+```
+
+## General Guidelines
+
+To keep React UI consistent and predictable the following guidelines should be observed:
+
+1. If component accepts the `children` prop it should be either required or the element
+   should return `null` when no children are provided.
+
+## Documenting
```

Comment:
This section is out of date. Please update it according to new stack

Reply:
fixed